content/robot/index.md

```diff
@@ -1,8 +1,8 @@
 ---
 linkTitle: Robot
-title:  Robot - FRC 2024
+title:  Robot - FRC 2025
 date:   2024-03-11T18:36:45-04:00
-draft:  false
+draft:  true
```

Comment:
Doing this will cause a 404 error for anybody who scans QR codes that are already out there, or otherwise goes to the old page.

How about adding a big image that links to the other archive until we have a blurb for the 2025 bot?

Reply:
... and also reset `draft: false` thus killing the possibility of a 404 error.